libs/metall/src/dice/metall/detail/file.hpp

```diff
@@ -449,6 +455,56 @@ inline bool copy_file_sparse_linux(const std::filesystem::path &source_path,
   METALL_ERROR("Unable to copy {} to {}", source_path.c_str(), destination_path.c_str());
   return false;
 }
+
+/**
+ * @brief Performs an accelerated, in-kernel copy from src to dst
+ * @param src source file descriptor
+ * @param dst destination file descriptor
+ * @param src_size size of source file as obtained by ::fstat(src)
+ * @return if the operation was successful
+ *
+ * @note this function only can theoretically do something
```

Comment:
I don't get the note

Reply:
I think `clone_file_range` is theoretically permitted to not be an accelerated copy. I.e. it is permitted to just be a regular dense copy.

But the note is not really relevant to anyone using this. I'm going to remove it.